pom.xml

```diff
@@ -66,10 +76,124 @@
         <dependency>
             <groupId>com.massivecraft.factions</groupId>
             <artifactId>Factions</artifactId>
-            <version>2.13.7</version>
+            <version>2.14.0</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.massivecraft.massivecore</groupId>
+            <artifactId>MassiveCore</artifactId>
+            <version>2.14.0</version>
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.gmail.nossr50.mcMMO</groupId>
+            <artifactId>mcMMO</artifactId>
+            <version>2.1.51-SNAPSHOT</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.github.TechFortress</groupId>
+            <artifactId>GriefPrevention</artifactId>
+            <version>16.7.1</version>
+        </dependency>
+
+        <!-- Add this yourself to your local Maven repo! -->
+        <dependency>
+            <groupId>towny</groupId>
+            <artifactId>towny</artifactId>
+            <version>0.93</version>
+        </dependency>
+
+        <!-- NMS -->
```

Comment:
Yes, I added all Spigot versions with NMS Source.
You just need to add a Maven profile with a property pointing to your spigot jars-folder. Then you can build this plugin easily yourself (this helps other users testing their PRs)

Reply:
Idk how i can add a maven profil in intellj.

Reply:
You dont have to add it in intellij, you need to add it in your maven configuration file. The maven documentation Shows you how to add a Profile, add env variables to it and Set the profile active.

Reply:
Could you provide a example to speed it up ?

Reply:
I am not home right now. I should be able to send you examples in a few hours

Reply:
Already found it, but have an issue with two repositories (see the other comment)